### ISSUE TITLE
"HDD_STATUS" in api.py editieren

### ISSUE THREAD

**Issue by technikamateur** [technikamateur/FreeMind#42]
- umbenennen in HDD_ERROR
- Es gibt keinen Status, wie FAILED oder SUCCESS, da das Backup einmal täglich den Server kontaktiert und der Wert nicht gespeichert wird. D.h. Wenn defekte HDD am BPI ausgetauscht wird, wird HDD Error nicht mehr gesendet. Ein Neustart des Masters reicht und der Fehler ist weg.

**Comment by vale981** [comment]
Ergibt doch sinn.
Der BPI will den HDD STATUS also bekommt er ihn.

**Comment by technikamateur** [comment]
Nein. Der BPI will den nicht. Er übergibt ihn.

**Comment by technikamateur** [comment]
Nur bei DO_BACKUP wird mitgeteilt, ob der Server antwortet dann mit "true" oder "false" (klein geschrieben).

**Comment by vale981** [comment]
Na dann. Vielleicht will der Pi in auch mal als warning uebergeben?

**Comment by technikamateur** [comment]
Stimmt. Na dann könnte er als Status WARNING und FAILED übertragen (für SMART Test nicht bestanden und Laufwerk ausgefallen).
Funktioniert eigentlich DO_BACKUP (sprich, das er alle Festplatten auf Onlinestatus (nicht SMART) überprüft?
Und schigg mal bitte keine Audio's. Bin doch auf Arbeit...

**Comment by vale981** [comment]
Ja

**Comment by technikamateur** [comment]
Wow. Einfach nur ja? Kein meckern? Keine Kritik?
